app/client/templates/single-app/single-app.scss

```diff
@@ -127,19 +145,23 @@ $app-container-padding-rl: 4.6rem;
           .button {
             display: inline-block;
             /* margin here pads the entire .top-section. */
-            margin-bottom: 0.5rem;
-
+            margin-bottom: 1rem;
+            margin-left: 0.5rem;
             height: 2.7rem;
             line-height: 2.7rem;
             font-size: 1.5rem;
-            padding-left: 1rem;
-            padding-right: 1rem;
+            padding: 0.4rem 1rem;
             background-color: inherit;
-            border-color: #e7daeb;
+            border-color: #cec2d2;
             border-radius: 0.5rem;
             border-style: solid;
             border-width: 1px;
             color: #783189;
+            @media (max-width: 460px) {
```

Comment:
It's slightly strange to have a `460px` breakpoint here and a `480px` breakpoint below. It's not _super_ important to me that we be super consistent, just saying it's slightly odd.

Not a show-stopper.